package.json

```diff
@@ -31,13 +31,13 @@
     "inputmask-core": "^2.1.1"
   },
   "peerDependencies": {
-    "react": "0.14.x"
+    "react": "15.x.x"
```

Comment:
in the `||` column. Minor versions are unlikely to introduce breaking changes and we by default support all future versions. The difference, IMO, will be when we drop support for older versions if newer versions make it difficult.